app-web/src/pages/index.js

```diff
@@ -135,6 +135,10 @@ export const resourceQuery = graphql`
           attributes {
             persona
           }
+          collection {
```

Comment:
GitHub is showing these in dark-read. This usually means a syntax error but I don't see any in the string.

Reply:
yeah i'm not sure why this is happening.. ill take a look into it